### PR TITLE
Issue #162 : Add tests for equals comparision in function as attribute

### LINKED ISSUE #162: False positive for named argument checks when using a == operator in an argument
```twig
{{- mdc_tab(
        'onboarding_program.menu.in_progress'|trans,
        path('training_program_list'),
        completed == false,
        'extra arg'
    ) -}}
```

This reports an error on the `=` sign in `completed == false` saying `ERROR There should be 0 space before the "=" in the named arguments list.` but this is not a named argument list

### PR DESCRIPTION
**Pull request by Ciloe**
Signed-off-by: Jimmy ESCRICH <jescrich@ruedesvignerons.com>

Closed #162

### CODE REVIEW

File: tests/Twig3FunctionalTest.php [hunk 1]
```diff
@@ -25,6 +25,10 @@ public function testExpressions($expression, $expectedViolation, array $expected
         $validator = new Validator();
 
         $violations = $validator->validate(new Official(3), $lexer->tokenize(new Source($expression, 'src', 'src.html.twig')));
+        if (count($validator->getCollectedData()[RegEngineRule::class]['unrecognized_expressions'] ?? []) > 0) {
+            var_dump($violations);
+            die;
```
Comment (stof) [inline tests/Twig3FunctionalTest.php]: this should be removed

**Review by stof — CHANGES_REQUESTED** [review]

File: tests/Twig3FunctionalTest.php [hunk 2]
```diff
@@ -25,6 +25,10 @@ public function testExpressions($expression, $expectedViolation, array $expected
...
+        }
```
Comment (Ciloe) [inline tests/Twig3FunctionalTest.php]:
```suggestion
```

**Comment by stof** [comment]
@Ciloe anything missing to finish that bugfix ?

**Comment by stof** [comment]
Hello @Ciloe. any chance to get this merged (and released) ?

**Comment by Ciloe** [comment]
Seen with @OwlyCode, we will merge soon

**Review by OwlyCode — APPROVED** [review]

**Pull request merged**